mtp_api/apps/transaction/api/bank_admin/views.py

```diff
@@ -66,12 +69,14 @@ def patch_processed(self, request, *args, **kwargs):
         try:
             return self.partial_update(request, *args, **kwargs)
         except Transaction.DoesNotExist as e:
+            transaction_ids = sorted(e.args[0])
+            logger.warn('Some transactions failed to update: [%s]' % ', '.join(map(str, transaction_ids)))
```

Comment:
There are similar scenarios in `CreditTransactions` and `UnlockTransactions` in api/cashbook/views.py which it might also be worth logging, if you're logging it here.